.github/workflows/cloud-tpu-ci-nightly.yml

```diff
@@ -0,0 +1,59 @@
+name: Cloud TPU nightly
+
+on:
+  schedule:
+    - cron: "0 14 * * *"  # daily at 7am PST
+  workflow_dispatch:  # allows triggering the workflow run manually
+
+# Should also be set to read-only in the project settings.
+permissions:
+  contents: read
+
+jobs:
+  cloud-tpu-test:
+    runs-on: v4-8
+    defaults:
+      run:
+        shell: bash -l {0}
+    strategy:
+      fail-fast: false  # don't cancel all jobs on failure
+      matrix:
+        python-version: ["3.10"]  # TODO(jakevdp): update to 3.11 when available.
+        jaxlib-version: [latest, nightly]
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install JAX test requirements
+        run: |
+          pip install -r build/test-requirements.txt
+      - name: Install JAX
+        run: |
+          pip uninstall -y jax jaxlib libtpu-nightly
+          if [ "${{ matrix.jaxlib-version }}" == "latest" ]; then
+            pip install .[tpu] \
+              -f https://storage.googleapis.com/jax-releases/libtpu_releases.html
```

Comment:
Oh unrelated to this PR, but I'd actually be interested in how you decided to set this up. We used to parse the GitHub releases page, but they changed the format, so now we make a hacky not-index and are looking for what the best long-term solution is: https://github.com/iree-org/iree/issues/9532

I see you're using `-f` *and* `--pre`

Reply:
Yeah let's take this offline :-)